declaring After=syslog.target is unnecessary by now because
syslog is socket-activated and will therefore be started when needed.
--- a/smartd.service.in
+++ b/smartd.service.in
@@ -1,7 +1,6 @@
 [Unit]
 Description=Self Monitoring and Reporting Technology (SMART) Daemon
 Documentation=man:smartd(8) man:smartd.conf(5)
-After=syslog.target
 
 [Service]
 EnvironmentFile=-/usr/local/etc/sysconfig/smartmontools
